layouts/partials/header.html

```diff
@@ -21,7 +21,7 @@
                     {{ end }}
                 {{ end }}
                 {{ $srcset := delimit ($.Scratch.Get "srcset") "," }}
-                <a {{ printf "href=%q" ("/" | relLangURL) | safeHTMLAttr }}><img class="avatar" src="/{{ .Site.Params.avatar }}" {{ printf "srcset=%q" $srcset | safeHTMLAttr }}></a>
+                <a {{ printf "href=%q" ("/" | relLangURL) | safeHTMLAttr }}><img class="avatar" src="{{ .Site.BaseURL  }}{{ .Site.Params.avatar }}" {{ printf "srcset=%q" $srcset | safeHTMLAttr }}></a>
```

Comment:
Wouldn't `.Site.BaseURL` also be needed as the target of the avatar link (the round image at the top of the page), as well as when clicking on the name right below it? Perhaps Hugo already writes the links correctly, never placed a site in a subdirectory.